src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/service/ReferralService.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.service
+
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto.DraftReferral
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Referral
+
+object ReferralService {
+  fun updateReferral(original: Referral?, update: DraftReferral): Referral? {
```

Comment:
`original` can be `Referral` without the nullable qualifier. we are checking for null at the call site (line 58 in ReferralController.kt)

Reply:
done in 82211c3015da4b02b8c2a6a035563e856c102969